### DISCUSSION TITLE
vuepress-plugin-github-linkify

### DISCUSSION THREAD

**Discussion by andrey-helldar** [Laravel-Lang/common#68] [General]
@luisprmat, @mohamedsabil83, hey!

In the example of the [plugin](https://github.com/TheDragonCode/vuepress-plugin-github-linkify) "before" and "after" I used links to your accounts, because just copied different lines for example. Do you mind it? Or is it better to replace the screenshots?

Repo: https://github.com/TheDragonCode/vuepress-plugin-github-linkify

![after](https://github.com/TheDragonCode/vuepress-plugin-github-linkify/blob/main/.github/images/after.png?raw=true)

**Comment by mohamedsabil83** [comment]
Go ahead from my side and great work by the way 👍🏽

**Reply by andrey-helldar** [reply]
شكراً لك 🤝

The result of the plugin's work can be seen here, for example: https://laravel-lang.com/referents.html

**Reply by luisprmat** [reply]
Great job! Go ahead!

**Reply by andrey-helldar** [reply]
@luisprmat, gracias 🤝

**Reply by mohamedsabil83** [reply]
Пожалуйста

btw, the links of the **ar** went to the **af**. just need to fix it.

**Reply by andrey-helldar** [reply]
@mohamedsabil83, oh, sorry. I copied the first folder link and forgot to change it. Thank you for your comment!
